pkgs/top-level/perl-packages.nix

```
@@ -10341,6 +10341,21 @@ let
     };
   };
 
+  LogJournald = buildPerlModule {
+    pname = "Log-Journald";
+    version = "0.30";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LK/LKUNDRAK/Log-Journald-0.30.tar.gz";
```

allow for bot to update
```suggestion
      url = "mirror://cpan/authors/id/L/LK/LKUNDRAK/Log-Journald-${version}.tar.gz";
```